From ce41aed2d663d6a7278b83bcdc2dc0adb65b9f5c Mon Sep 17 00:00:00 2001
From: Tiago Medicci <tiago.medicci@espressif.com>
Date: Wed, 13 Nov 2024 14:23:34 -0300
Subject: [PATCH 08/14] declare struct timeval

Otherwise, build will fail due to redefinition of _PyTime_FromTimeval

Signed-off-by: Tiago Medicci <tiago.medicci@espressif.com>
---
 Include/internal/pycore_time.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Include/internal/pycore_time.h b/Include/internal/pycore_time.h
index 205ac5d3781..c30e07f4b4a 100644
--- a/Include/internal/pycore_time.h
+++ b/Include/internal/pycore_time.h
@@ -58,9 +58,7 @@ extern "C" {
 #endif
 
 
-#ifdef __clang__
 struct timeval;
-#endif
 
 #define _SIZEOF_PYTIME_T 8
 
-- 
2.50.0

